src/Components/DishItems/DishMenu.js

```diff
@@ -0,0 +1,30 @@
+import DishCategoryRow from "./DishCategoryRow";
+import "./DishMenu.css";
+import DishRow from "./DishRow";
+
+const getDishTemplate = function (items) {
+  const rows = [];
+  Object.keys(items).forEach((category) => {
+    const currItem = items[category];
+    rows.push(
+      <DishCategoryRow
+        key={category}
+        heading={category}
+        subtitle={currItem.length}
+      >
+        {currItem.map((item) => {
```

Comment:
Can make separate fn/component

Reply:
It's a separate component, it's getting rendered as its children